src/wallet/spend.cpp

```diff
@@ -83,7 +83,14 @@ TxSize CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *walle
     return CalculateMaximumSignedTxSize(tx, wallet, txouts, coin_control);
 }
 
-void AvailableCoins(const CWallet& wallet, std::vector<COutput>& vCoins, const CCoinControl* coinControl, const CAmount& nMinimumAmount, const CAmount& nMaximumAmount, const CAmount& nMinimumSumAmount, const uint64_t nMaximumCount)
+void AvailableCoins(const CWallet& wallet,
+                    std::vector<COutput>& vCoins,
+                    const CCoinControl* coinControl,
+                    const CAmount& nMinimumAmount,
+                    const CAmount& nMaximumAmount,
+                    const CAmount& nMinimumSumAmount,
+                    const uint64_t nMaximumCount,
+                    std::vector<COutput>* vUnconfCoins)
```

Comment:
In b2846445135f6d6b7902672cb2202dc75088a200 "wallet: add CoinControl filtering capabilities for coins that exceed mempool restrictions."

nit: For new variables, we prefer snake_case, without the prefix for type. So this could be just `unconf_coins`.